supported-standards/lens-metadata-standards/publication/event/1.0.0/README.md

```diff
@@ -0,0 +1,128 @@
+# Event 1.0.0
+
+## JSON schema
+
+The JSON schema can be found [here](./schema.json).
+
+## TypeScript
+
+References:
+
+- [MarketplaceMetadata](../../shared-ts-interfaces/marketplace-metadata.ts)
+- [MetadataAttribute](../../../shared-ts-interfaces/metadata-attribute.ts)
+- [MetadataCommon](../../shared-ts-interfaces/metadata-common.ts)
+- [PublicationMainFocus](../../shared-ts-interfaces/publication-main-focus.ts)
+- [MediaVideo](../../shared-ts-interfaces/media/media-video.ts)
+- [MediaImage](../../shared-ts-interfaces/media/media-image.ts)
+- [MediaAudio](../../shared-ts-interfaces/media/media-audio.ts)
+
+```ts
+interface EventMetadata extends MarketplaceMetadata {
+  /**
+   * The JSON schema for the event
+   */
+  $schema: "https://raw.githubusercontent.com/lens-protocol/LIPs/feat/metadata-standards/lens-metadata-standards/publication/event/1.0.0/schema.json";
+
+  /**
+   * The metadata details for the event
+   */
+  lens: EventMetadataDetails;
+
+  /**
+   * If you use an appId and you only want your app to be able to publish under it,
+   * you will need to sign the `lens` object with your server private key
+   * and populate the signature here. We will need your public key to verify this.
+   * If the signature does not match what we have in our mappings it will not be surfaced
+   * in the API.
+   */
+  signature?: Signature;
+}
+
+interface EventMetadataDetails extends MetadataCommon {
+  /**
+   * The title of the event
+   */
+  title: string;
+
+  /**
+   * Where the event is in clear text
+   */
+  location: string | URI;
```

Comment:
Pretty interesting...I didn't know about Activity Streams.
 
 Are you suggesting these also as replacement for `geographic` (few lines blow)?

Reply:
Yes, I think location and geographic should be 1 property as you usually don't need both. 
So location would be of type `position` -OR- `address`

Reply:
@wslyvh some movement in this direction here: https://github.com/lens-protocol/metadata/pull/15/commits/e9808e654adbbb12ab1256302a71c7bddb98e446

Will update this PR soon.

Reply:
LGTM

I was thinking that you might need another option for location in case it's an online event.

One option could be to allow a `string` next to `geoUri` -or- `address`, which could be used to link to the meeting/video call. Not sure how feasible this would be, but Meetup only shows that link with people who actually RSVP-ed.

Alternatively, a separate boolean to toggle as `online` event and just use the `links` property could work.

Reply:
Re: online event... the `location: string | URI` is meant to support that... but also using a `location` as `string and adding `links` is another valid usage.

Re:
> Meetup only shows that link with people who actually RSVP-ed.

This seems a feature for token-gated. The whole publication is encrypted until some criteria are met by the "reader".
The logic currently allows to check for NFT ownership, ERC20 ownership, collected publication, follow criteria, profile ownership, EOA ownership, and can be extended but minimal effort.

I can see something like you describe implementable through token-gate in a couple of ways using existing capabilities depending on the event organizer's needs.

Reply:
Ah, I assumed the location type would be replaced by `GeoURI | Address`. I guess `GeoURI | address | string` and using `links` would work for most use-cases.

RE token-gating, that makes sense! Will check it out :)